src/react-imgix.js

```diff
@@ -252,3 +279,54 @@ export default class ReactImgix extends Component {
     return React.createElement(_component, childProps, _children);
   }
 }
+
+const DEPRECATED_PROPS = ["auto", "crop", "entropy", "faces", "fit"];
+const deprecateProps = WrappedComponent => {
+  const WithDeprecatedProps = props => {
+    const imgixParams = {
+      ...(props.customParams || {}),
+      ...props.imgixParams
+    };
+    const propsWithOutDeprecated = {
+      ...props,
+      imgixParams
+    };
+    DEPRECATED_PROPS.forEach(deprecatedProp => {
+      warning(
+        !(deprecatedProp in props),
+        `The prop '${deprecatedProp}' has been deprecated and will be removed in v9. Please update the usage to <Imgix imgixParams={{${deprecatedProp}: value}} />`
```

Comment:
Following my above comment about not encouraging users to pass in `{ imgixProps: { entropy: true } }`, this warning will need to be updated.